client/coral-plugin-commentbox/CommentBox.js

```diff
@@ -68,6 +101,15 @@ class CommentBox extends Component {
     this.setState({body: ''});
   }
 
+  addCommentHooks = (hooks = {}) => {
+    if (typeof hooks === 'object') {
```

Comment:
I think it's smart but it's also a bit confusing. But I agree that we could provide `register` and `unregister` methods.